server/src/main/java/io/druid/client/cache/Cache.java

```diff
@@ -32,6 +32,13 @@
 {
   public byte[] get(NamedKey key);
   public void put(NamedKey key, byte[] value);
+
+  /**
+   * Resulting map should not contain any null values (i.e. cache misses should not be included)
```

Comment:
@himanshug well, it's not technically required, but would make the code both more complex and slower if we have to check the value of each key in the map in order to filter them out in order to do a bulk L2 request.